src/esssans/isis/mantidio.py

```diff
@@ -21,6 +19,24 @@
 )
 from .io import CalibrationFilename, Filename, FilePath
 
+try:
+    import mantid.api as _mantid_api
+    import mantid.simpleapi as _mantid_simpleapi
+    from mantid.api import MatrixWorkspace
+except ModuleNotFoundError:
+    # Catch runtime usages of Mantid
+    class _MantidFallback:
+        def __getattr__(self, name: str) -> NoReturn:
+            raise RuntimeError(
```

Comment:
Do we need to update the exception type in the Zoom notebook?

Reply:
No because it imports mantid explicitly.

But I was actually wondering if it makes sense to raise `ImportError` here. Is there a guideline on this?

Reply:
Let us go with `ImportError`?